tests/zero-downtime-scaling/results/1.3.0/1.3.0.md

```diff
@@ -0,0 +1,474 @@
+- NGINX error/warn logs: none
+- NGINX access logs: 2 499s when accessing tea over HTTPS
+
+### Scale Up Abruptly
+
+HTTP wrk output
+
+```text
+Running 2m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.84ms    1.51ms  36.50ms   93.84%
+    Req/Sec   596.46    156.20     1.03k    74.51%
+  Latency Distribution
+     50%    1.50ms
+     75%    1.99ms
+     90%    2.76ms
+     99%    8.02ms
+  36590 requests in 30.82s, 13.04MB read
+  Socket errors: connect 0, read 1, write 0, timeout 0
+Requests/sec:   1187.24
+Transfer/sec:    433.27KB
+```
+
+HTTP Graph
+
+![abrupt-up-http.png](25-node/abrupt-up-http.png)
+
+HTTPS wrk output
+
+```text
+Running 2m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.88ms    1.78ms  49.07ms   95.13%
+    Req/Sec   588.31    158.25     0.99k    70.50%
+  Latency Distribution
+     50%    1.51ms
+     75%    2.02ms
+     90%    2.76ms
+     99%    8.65ms
+  37739 requests in 32.22s, 13.19MB read
+Requests/sec:   1171.20
+Transfer/sec:    419.06KB
+```
+
+HTTPS Graph
+
+![abrupt-up-https.png](25-node/abrupt-up-https.png)
+
+Logs:
+
+- NGF error log: none
+- NGINX error/warn logs: none
+- NGINX access logs: no non-200s
+
+### Scale Down Abruptly
+
+
+HTTP wrk output
+
+```text
+Running 2m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.51ms    1.30ms  55.34ms   94.11%
+    Req/Sec   722.71    201.09     1.15k    62.09%
+  Latency Distribution
+     50%    1.18ms
+     75%    1.62ms
+     90%    2.31ms
+     99%    6.27ms
+  88110 requests in 1.02m, 31.40MB read
+  Socket errors: connect 0, read 1, write 0, timeout 0
+Requests/sec:   1438.52
+Transfer/sec:    524.97KB
+```
+
+HTTP graph
+
+![abrupt-down-http.png](25-node/abrupt-down-http.png)
+
+HTTPS wrk output
+
+```text
+Running 2m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.57ms    1.11ms  36.38ms   92.86%
+    Req/Sec   683.88    178.63     1.05k    64.72%
+  Latency Distribution
+     50%    1.26ms
+     75%    1.69ms
+     90%    2.38ms
+     99%    6.18ms
+  88825 requests in 1.09m, 31.04MB read
+  Socket errors: connect 0, read 1, write 0, timeout 0
+Requests/sec:   1361.17
+Transfer/sec:    487.04KB
+```
+
+HTTPS graph
+
+![abrupt-down-http.png](25-node/abrupt-down-http.png)
+
+Logs
+
+- NGF error logs:
+    1 error: {"error":"leader election lost", "level":"error", "msg":"error received after stop sequence was engaged", "stacktrace":"sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).engageStopProcedure.func1
+	pkg/mod/sigs.k8s.io/controller-runtime@v0.18.3/pkg/manager/internal.go:499", "ts":"2024-06-05T18:05:43Z"}
```

Comment:
```suggestion
    1 error: ```{"error":"leader election lost", "level":"error", "msg":"error received after stop sequence was engaged", "stacktrace":"sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).engageStopProcedure.func1
	pkg/mod/sigs.k8s.io/controller-runtime@v0.18.3/pkg/manager/internal.go:499", "ts":"2024-06-05T18:05:43Z"}```
```

Should we wrap these errors in `code`?